### ISSUE TITLE
Feature and Bug Reports

### ISSUE THREAD

**Issue by rivermont** [rivermont/spidy#19]
Calling any passers-by to take a moment to submit feature requests or bugs, no matter how small!

**Comment by michellemorales** [comment]
Hey! Love your project. I see your `gui.py` says its under development. What's the status on that? Are you still interested in having a GUI for spidy so users don't need to know command line? I'd love a reason to work with [TKInter](https://wiki.python.org/moin/TkInter)

**Comment by rivermont** [comment]
Yeah, the original plan was to have a tkinter GUI to run the crawler, but we couldn't figure out threading and then school started :sweat_smile:. `gui.py` is the remnants of the attempt, but it never actually worked. The actual Tkinter code starts on [line 487](https://github.com/rivermont/spidy/blob/master/gui.py#L487); most everything else is old crawler code copied in. I'll make a new section in the docs for the GUI, there's a lot of thoughts.

**Comment by michellemorales** [comment]
Ok awesome. I'd love to take a crack at it :smiley:

**Comment by jmunson** [comment]
Is it possible to ignore TLS errors?

**Comment by ijf8090** [comment]
Nice utility. Thank you.

 What if I want to crawl a site that requires a log in. Ideally I would pass Username, Password as command line parameters in a shell script that I am using to QA my site.

Regards
ijf8090
